1.9/administration/debugging/quickstart.md

```diff
@@ -0,0 +1,208 @@
+
+    ```bash
+    $ dcos task exec <task_id> chmod a+x hello-world.sh
+    ```
+1. Run the script inside of the container:
+
+    ```bash
+    $ dcos task exec <task_id> ./hello-world.sh
+    ```
+
+    The output should look similar to this:
+    
+    ```bash
+    Hello World
+    ```
+
+# Launch a long running interactive Bash session
+
+In this example, a long running [job](/docs/1.9/usage/jobs/) is launched by using the `dcos job run` command, and the `dcos task exec` command is used to launch an interactive Bash shell inside the container of that job.
+
+1.  Deploy and run a job with the DC/OS CLI:
+
+    1.  Create the following job definition and save as `my-job.json`. This specifies a sleep job that runs for `10000000` seconds.
+    
+        ```bash
+        {
+          "id": "my-job",
+          "labels": {},
+          "run": {
+            "artifacts": [],
+            "cmd": "sleep 100000000",
+            "cpus": 0.01,
+            "disk": 0,
+            "env": {},
+            "maxLaunchDelay": 3600,
+            "mem": 32,
+            "placement": {
+              "constraints": []
+            },
+            "restart": {
+              "policy": "NEVER"
+            },
+            "volumes": []
+          }
+        }
+        ```
+    
+    1.  Deploy the job with this CLI command:
+    
+        ```bash
+        $ dcos job add my-job.json
+        ```
+        
+    1.  Verify that the job has been successfully deployed:
+    
+        ```bash
+        $ dcos job list
+        ID              DESCRIPTION  STATUS   LAST SUCCESFUL RUN        
+        my-job               Unscheduled         None
+        ```
+
+    1.  Run the job:
+    
+        ```bash
+        $ dcos job run my-job
+        ```
+
+1.  Get the task ID of the job with this CLI command:
+
+    ```bash
+    $ dcos task
+    ```
+    
+    The output should look similar to this: 
+    
+    ```bash
+    NAME                                HOST       USER  STATE  ID                                                                       
+    20161209183121nz2F5.my-job    10.0.2.53  root    R    <task_id>
+    ```
+
+1.  Launch a process inside of the container with the task ID (`<task_id>`) specified and attach a TTY to it. This will launch an interactive Bash session.
+
+    ```bash
+    $ dcos task exec --interactive --tty <task_id> bash
+    ```
+    
+    You should now be inside the container running an interactive Bash session.
+    
+    ```bash
+    root@ip-10-0-2-53 / #
+    ```
+    
+    **Tip:** You can use shorthand abbreviations `-i` for `--interactive` or `-t` for `--tty`. Also, only the beginning unique characters of the `<task_id>` are required. For example, if your task ID is `exec-test_20161214195` and there are no other task IDs that begin with the letter `e`, this is valid command syntax: `dcos task exec -i -t e bash`. For more information, see the CLI command [reference](/docs/1.9/usage/cli/command-reference/).
+    
```

Comment:
Since this is the last thing in the section now, maybe move this tip to the very bottom so as not to break the flow?